style/rails/README.md

```diff
@@ -33,6 +33,7 @@ Migrations
 
 * Set an empty string as the default constraint for non-required string and text
   fields. [Example][default example].
+* Set an explicit `on_delete` behaviour for foreign keys.
```

Comment:
Maybe link to http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_foreign_key docs?